sdk/nodejs/iam/documents.ts

```diff
@@ -68,14 +70,14 @@ export interface PolicyStatement {
     NotPrincipal?: Principal;
 
     // Include a list of actions that the policy allows or denies.
-    Action?: string | string[];
+    Action?: Input<string> | Input<string>[];
     // Include a list of actions that are not covered by this policy.
-    NotAction?: string | string[];
+    NotAction?: Input<string> | Input<string>[];
 
     // A list of resources to which the actions apply.
-    Resource?: string | string[];
+    Resource?: Input<string> | Input<string>[];
```

Comment:
this one is crazier in that it should also be wrapped with a higher level `Input<...>`  (SAme with the guys above it).